CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.79.6
```

Comment:
We actually haven't released 1.79.5 yet, so you might as well put this in there too.

Reply:
oks, I will do that then :)